### DISCUSSION TITLE
прикол с аватаркой

### DISCUSSION THREAD

**Discussion by nikersond0** [OpenVK/openvk#1057] [Q&A]
![image](https://github.com/openvk/openvk/assets/117586864/19ee9a3f-0802-42f9-8298-376a1def2d83)
посмотрите на исходную фотку (большая) и на миниатюру. чет она повернулась на 90 градусов, странно

**Comment by Lumaeris** [comment]
Подозреваю, что это просто техническая неисправность в одной из стандартных библиотек PHP. Могу предложить открыть фотографию в Paint и пересохранить без изменений, желательно в другом формате.
